src/Components/Pages/TabsPage.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { Link, useParams } from 'react-router-dom';
+import classnames from 'classnames';
+
+export const TabsPage = ({ tabs }) => {
+    const { tabId } = useParams();
+
+    const currentTab = useMemo(() => (
+        tabs.find(tab => tab.id === tabId)
+      ), [tabId]);
+
+      return (
+        <>
+        <h1 className="title">Tabs Page</h1>
+  
+        <div>
+          <ul className = "nav nav-tabs">
+            {tabs.map(tab => (
+              <li
+                key={tab.id}
+                className={classnames('nav-link',
+                  {'active': tab.id === tabId,
+                })}
```

Comment:
```suggestion
```
Instead of using this, you should use `<NavLink>` with `className` and `activeClassName` attributes. Let's fix this